network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -990,36 +992,38 @@ class InterferesQuery(TwoNetworkConfigsQuery):
     """
 
     def exec(self):
+        base_config = self.config2
+        other_config = self.config1
```

Comment:
I'm not sure these aliases really help readability.
Later in the code `name1` and `name2` are used, and it's hard to remember which refer to which